src/server/project.ts

```diff
@@ -198,13 +198,13 @@ namespace ts.server {
             return hasOneOrMoreJsAndNoTsFiles(this);
         }
 
-        public static resolveModule(moduleName: string, initialDir: string, host: ServerHost, log: (message: string) => void): {} | undefined {
+        public static resolveModule(moduleName: string, initialDir: string, host: ServerHost, log: (message: string) => void, logErrors?: (message: string) => void): {} | undefined {

```

Comment:
how bad is an API break? it would be less confusing just to require `logErrors`.

Reply:
I am not sure who we will break so its better to just have it optional I think